### PR TITLE
Feature/more languages, i.e. manage languages with crowdin

### PR DESCRIPTION
**Pull request by jonasheinrich**
https://community.openproject.org/work_packages/19745
https://community.openproject.org/work_packages/14922

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://jonasheinrich-feature-more-languages.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://jonasheinrich-feature-more-languages.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by NobodysNightmare** [comment]
Great stuff!

Do you know the quality of the current german translation on crowdin?
Is it the same as was in the code? Or were those already community translated strings which we have to re-verify.

Just asking because german was the only language that we officially verified and on crowdin only few translations are currently officially verified.

File: frontend/app/openproject-app.js [hunk 1]
```diff
@@ -30,7 +30,6 @@ var I18n = require('./vendor/i18n');
 
 // standard locales
 I18n.translations.en = require("locales/js-en.yml").en;
-I18n.translations.de = require("locales/js-de.yml").de;
```
Comment (NobodysNightmare) [inline frontend/app/openproject-app.js:33]: In the corresponding plugin PRs you are also removing the standard english locale, e.g. here: https://github.com/finnlabs/openproject-backlogs/pull/142

Probably I am missing a point here, but I'd expect those changes to look similar...

Comment (jonasheinrich) [inline frontend/app/openproject-app.js:33]: The reason for this difference is that the locales (except english) for the core are saved in OpenProject-Translations and the locales for the plugins are saved in the plugins themselves. Thus, the plugins load all locales and the core only loads english (OpenProject-Translation loads the rest).

**Comment by ulferts** [comment]
@jonasheinrich please rebase to get rid of the spec failure.

**Comment by ulferts** [comment]
@jonasheinrich and apparently you need to rebase anyway to reestablish mergeability

**Comment by ulferts** [comment]
@NobodysNightmare brings up a good point, we should compare the locale files you remove here and the one on crowdin. We spent some time with the core's local file so we should not simply throw it away if there is still something in there we can use.

**Comment by tessi** [comment]
\o/ for the attempt to use crowdin

**Comment by NobodysNightmare** [comment]
`\o/` could either be someone raising his arms (like _hooray_)... or it could be a 1337 way of writing `lol` :smile_cat:

**Comment by tessi** [comment]
@NobodysNightmare funny that it has those meanings to you. But it was, indeed, a 'hooray' :)

**Comment by floriank** [comment]
I always read `\o/` as :metal:

**Comment by NobodysNightmare** [comment]
This PR is still WIP according to Jonas.

**Comment by jonasheinrich** [comment]
Closed in favor of #3211

**Pull request closed without merging**